haskell-indexer-pipeline-ghckythe-wrapper/src/Language/Haskell/Indexer/Args.hs

```diff
@@ -109,10 +121,21 @@ flagParser = Flags
              <> help ("Overrides the preprocessor binary, but keeps it's "
                       ++ "options. Note: other tools can still be overriden "
                       ++ "by passing the regular -pgmX GHC options.")))
+     <*> optional (strOption
+            ( long "output"
+            <> short 'o'
+            <> metavar "PATH"
```

Comment:
Not sure what metavar is used for, but should this be more specific? Like INDEX_OUT_PATH or such.